app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageActionsBox.kt

```diff
@@ -165,7 +165,7 @@ private fun AddMentionAction(isSelected: Boolean, addMentionAction: () -> Unit)
 private fun PingAction(onPingClicked: () -> Unit) {
     WireSecondaryIconButton(
         onButtonClicked = onPingClicked,
-        clickBlockParams = ClickBlockParams(blockWhenSyncing = true, blockWhenConnecting = true),
+        clickBlockParams = ClickBlockParams(blockWhenSyncing = false, blockWhenConnecting = false),
```

Comment:
Sending and editing messages are now blocked when syncing, I don't see any reason now why it shouldn't work fine during syncing as we mainly had issues with "connecting" state, so when offline, but it's always better to make sure if that's fine, and if so maybe we could also enable sending and editing messages when syncing. 😄